src/03-modality-agnostic-files.md

```diff
@@ -467,6 +467,8 @@ and a guide for using macros can be found at
    }
 ) }}
```

Comment:
in the table above, I would expect the entry for `acq_time__scans` to contain the text "This is a TEST". However, it contains: `{{ MACROS___make_path_relative() }}`

Reply:
see: https://bids-specification--1056.org.readthedocs.build/en/1056/03-modality-agnostic-files.html#scans-file